fe/fe-core/src/main/java/org/apache/doris/qe/QeService.java

```diff
@@ -56,6 +62,12 @@ public void start() throws Exception {
             LOG.error("mysql server start failed");
             System.exit(-1);
         }
+        if (flightsqlPort != -1) {
+            this.flightSqlService = new FlightSqlService(flightsqlPort);
+            if (!flightSqlService.start()) {
```

Comment:
add  log here, so that we could not flight server not started

Reply:
fixed